packages/backend/server/src/__tests__/copilot.e2e.ts

```diff
@@ -88,6 +89,10 @@ test.before(async t => {
       WorkspaceModule,
       CopilotModule,
     ],
+    tapModule: m => {
+      // use real JobQueue for testing
+      m.overrideProvider(JobQueue).useClass(JobQueue);
```

Comment:
@darkskygit  This e2e test case may need some adjustments. If it relies on the queue driver, it should be unpacked. The e2e test should only verify that the queue add is successful.